docs/application-migration/learning-path-for-developers.md

```diff
@@ -0,0 +1,23 @@
+## Learning Path for Developers
+
+1. Twelve-Factor App
+   1. Video Session: [12 Factors to Container Success](https://developers.redhat.com/blog/2017/06/22/12-factors-to-cloud-success) (1 hour)
+   2. [Official Documentation](https://12factor.net/)
+2. Learning Path on Containers:
+   1. A Practical [Introduction to Container](https://developers.redhat.com/blog/2018/02/22/container-terminology-practical-introduction#) Terminology
+   2. Container Fundamentals
+   3. Free Self-Paced Training [Running Containers with Red Hat Technical Overview](https://www.redhat.com/en/services/training/rh065-running-containers-red-hat-technical-overview) (1 hour)
+3. Learning Path on Kubernetes/OpenShift :
+   1. OpenShift Fundamentals:
+      1. Free Self-Paced Training: [Containers, Kubernetes and Red Hat OpenShift Technical Overview](https://www.redhat.com/en/services/training/do080-deploying-containerized-applications-technical-overview) (1 hour)
+      2. Free Self-Paced Training [Getting started with OpenShift](https://developers.redhat.com/learn/openshift/develop-on-openshift) (Optional)
+      3. [OpenShift Official Documentation](https://docs.openshift.com/container-platform/4.10/web_console/web-console-overview.html)
+      4. [Kubernetes Official Documentation](https://kubernetes.io/docs/concepts/overview/)
+4. Learning Path on CI/CD
+   1. What’s CI/CD
+      1. [https://www.redhat.com/en/topics/devops/what-cicd-pipeline](https://www.redhat.com/en/topics/devops/what-cicd-pipeline)
```

Comment:
When referring to link you should either substitute it for text or use `<>`.

```suggestion
      1. [What is a CI/CD pipeline?](https://www.redhat.com/en/topics/devops/what-cicd-pipeline)
```

or 

```suggestion
      1. <https://www.redhat.com/en/topics/devops/what-cicd-pipeline>
```